packages/app/components/stack-modal/StackModal.tsx

```diff
@@ -50,6 +52,17 @@ interface StackModalProps extends ModalBaseProps {
   refetchStacks: () => void;
 }
 
+type Content = {
+  title: string;
+  description: string;
+  button: {
+    action: "primary" | "secondary";
+    text: string;
+  };
+};
+
+export const txEplorerLink = (tx: string) => `https://gnosisscan.io/tx/${tx}`;
+
 export const addressExplorerLink = (address: string) =>
   `https://gnosisscan.io/address/${address}#tokentxns`;
```

Comment:
If we are using this elsewhere we might move this to @/model

Reply:
removed this method, as we have one on `utils/transaction` file